src/main/java/com/artipie/Layout.java

```diff
@@ -97,12 +97,9 @@ public boolean hasDashboard() {
     }
 
     /**
-     * Flat layout. Consists of two parts.
+     * Org layout. Consists of one part.
      *
```

Comment:
@andreoss should be `two parts`

Reply:
@genryxy Done

Reply:
@andreoss `Org` has two parts, whereas `Flat` has one part

Reply:
@genryxy Fixed

Reply:
But two parts of what exaclty? This doc can be improved futher